src/test/regress/sql/pg17.sql

```diff
@@ -228,3 +228,42 @@ ALTER EXTENSION citus DROP ACCESS METHOD fake_am;
 -- End of testing specifying access method on partitioned tables.
 
 DROP SCHEMA pg17 CASCADE;
+
+-- Correlated sublinks are now supported as of PostgreSQL 17, resolving issue #4470.
+CREATE SCHEMA local_table_join_pg17;
+SET search_path TO local_table_join_pg17;
+-- Enable DEBUG-level logging to capture detailed execution plans
+SET client_min_messages TO DEBUG1;
+SET log_min_messages TO DEBUG1;
+SET citus.worker_min_messages TO DEBUG1;
```

Comment:
We don't need a new schema here, we can just use the pg17 one of the test, and drop it afterwards.

Also I believe setting client_min_messages is enough

```suggestion
-- Correlated sublinks are now supported as of PostgreSQL 17, resolving issue #4470.
-- Enable DEBUG-level logging to capture detailed execution plans
SET client_min_messages TO DEBUG1;
```